packages/deployment-service-ui/package.json

```diff
@@ -32,6 +32,7 @@
     "react-ga": "^2.7.0",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
+    "shleemy": "^1.2.2",
```

Comment:
Lol

Reply:
ganna add it everywhere until it gets 2000 downloads a week :joy: